bin/hammer

```diff
@@ -52,12 +52,16 @@ end
 
 CFG_PATH_LOCAL = ['./config/']
 
+HammerCLI::Settings.load_from_paths CFG_PATH_LOCAL
+
 if preparser.config
-  CFG_PATH_LOCAL.unshift preparser.config
+  if File.file? preparser.config
+    HammerCLI::Settings.load_from_file preparser.config
```

Comment:
Does it work with tilde too? It seems that `load_from_file` does not expand paths.